Integrate-Kobiton.md

```diff
@@ -0,0 +1,133 @@
+
+2. Copy contents of `sample` folder into your repository.
+
+## B. Setup
+### 1. Getting required parameters
+In order to execute test(s) on Kobiton, these three must-have parameters must be provided : Kobiton Username, API key and desired capabilities.
+
+#### 1.1 Getting Kobiton Username and API Key
+
+To get your Kobiton Username and API Key, follow instruction at `IV. Configure Test Script for Kobiton` section on [our blog article](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/).
+
+#### 1.2 Getting Kobiton device desired capabilities
+
+To get Kobiton device desired capabilities, follow these steps below :
+
+1. Go to [https://portal.kobiton.com/login](https://portal.kobiton.com/login) and login with your Kobiton account.
+2. Click **"Devices"** at the top of the window.
+
+![Devices button](./assets/devices-button.png)
+
+3. Hover over the device you want to run your test script on, click the gear button.
+
+![Automation Settings button](./assets/gear-button.png)
+
+4. In the popup window, you can choose the language of your script, application testing type, etc. In this example, we will be using NodeJS, therefore, change `Language` section to `NodeJS`. Then, change the value of `App Type` to `Hybrid/Native from Apps`. The desired capabilities and Kobiton server web driver configuration code will be generated and displayed in the right pane of the popup (marked **blue**).
+
+![Automation Settings](./assets/desired-caps.png)
+
+5. Write down the generated code as we will be using it later.
+
+### 2. Configuring automation test script
+
+#### 2.1 Setting Username and API key
+
+In your project configuration in CircleCI, add these two environment variables :
+- KOBITON_USERNAME - Your Kobiton's username.
+- KOBITON_API_KEY - Your Kobiton's API Key.
+
+Your `Environment Variables` window should look like this
+
+![alt text](./assets/env-variables-after-add.png)
+
+#### 2.2 Setting Desired Capabilities
+
+1. Open the automation test script
+
+2. Replace `desiredCaps` value with the one taken above. The result should look like this
+
+```javascript
+const desiredCaps = {
+  sessionName:        'Automation test session',
+  sessionDescription: '', 
+  deviceOrientation:  'portrait',  
+  captureScreenshots: true, 
+  app:                '<YOUR_APP_URL>', 
+  deviceGroup:        'KOBITON', 
+  deviceName:         '<DEVICE_NAME>',
+  platformVersion:    '<PLATFORM_VERSION>',
+  platformName:       '<PLATFORM_NAME>' 
+}
+```
+
+> More information about Desired Capabilities can be found in https://docs.kobiton.com/automation-testing/desired-capabilities-usage/
+
+> You can visit [our blog](https://kobiton.com/blog/automation-web-appium-kobiton-nodejs/) for details on how to set up an automation test script.
+
+## C. Automation Test Execution
+### 1. Executing the automation test
+
+Simply push all changes to your repository and CircleCI will automatically execute the test script.
```

Comment:
What are all changes? Please tell clearer.

Reply:
How will CircleCI execute the test script automatically? Please describe a little bit about `.circleci/config.yml`, where is the place to put it in and how it works (stages, commands).